pkg/libinit/proc_linux.go

```diff
@@ -62,6 +96,57 @@ func linuxDefault(c *exec.Cmd) {
 	}
 }
 
+func Raw(r *os.File) error {
```

Comment:
did you have to break all this out? There is a Raw function in one of the packages? 
If there is not a raw function, can you consider adding this to a package, so others can use it? It's a shame to have this only in libinit.

Reply:
I agree to move this out. Is was copied into the libinit package for the initial PoC but for this PR it should be moved to an appropriate place. Will do so.